### DISCUSSION TITLE
视频无法播放

### DISCUSSION THREAD

**Discussion by wule1433** [WatsonYao/nian_app#76] [Q&A] [answered]
再点击视频功能键后，在直接弹出的选项中选择视频，在发布前视频是可播放的，但发布后无法播放。     要发布后也能播放需要前往具体文件夹，选择视频才可，希望可以解决这个问题。

**Comment by WatsonYao** [comment]
nian没有视频播放的功能，需要跳转第三方视频播放软件来播放。两个思路，一个是查看视频源文件是否还在，一个是替换不同的视频播放app。

**Reply by llwy142086** [reply]
题主的问题我也遇到过！视频在发布前，可以完美跳转本地播放器进行播放，但是在发布后跳转第三方软件无法播放，源文件都存在，并且各种视频播放器都试过了，都不能正常播放。

**Reply by WatsonYao** [reply]
方便录屏看下你们的操作吗？这边几个测试设备都是正常的，你们添加的时候是通过点击添加视频按钮添加的吧？

**Reply by WatsonYao** [reply]
录屏不方便放这里的话，可以发邮件到 yaoyuan131617@gmail.com

**Reply by llwy142086** [reply]
录屏已经发到邮箱了，麻烦你看一下啦。

**Comment by WatsonYao** [comment, marked as answer]
帮助文档已新增相关QA。
